spec/meilisearch/index/documents_spec.rb

```diff
@@ -28,14 +28,14 @@
       it 'adds JSON documents (as a array of documents)' do
         documents = <<JSON
         [
-          { "objectId": 123,  "title": "Pride and Prejudice",                    "comment": "A great book" },
-          { "objectId": 456,  "title": "Le Petit Prince",                        "comment": "A french book" },
-          { "objectId": 1,    "title": "Alice In Wonderland",                    "comment": "A weird book" },
-          { "objectId": 1344, "title": "The Hobbit",                             "comment": "An awesome book" },
-          { "objectId": 4,    "title": "Harry Potter and the Half-Blood Prince", "comment": "The best book" }
+          { "myId": 123,  "title": "Pride and Prejudice",                    "comment": "A great book" },
+          { "myId": 456,  "title": "Le Petit Prince",                        "comment": "A french book" },
+          { "myId": 1,    "title": "Alice In Wonderland",                    "comment": "A weird book" },
+          { "myId": 1344, "title": "The Hobbit",                             "comment": "An awesome book" },
+          { "myId": 4,    "title": "Harry Potter and the Half-Blood Prince", "comment": "The best book" }
```

Comment:
If you really want to make this test relevant, you might want to use a primary key that does not contain `id` since MeiliSearch automatically infers it in this case. For example "refNumber"

Reply:
@curquiza mm.. got it. I'll do that.